README.md

```diff
@@ -103,7 +103,7 @@ The first time playwright is used, let it download the tools it needs with:
 playwright install
 ```
 
-*Start the dev server in the background*, then execute tests with
+*Start the docker container with the local config*, then execute tests with
```

Comment:
Does this mean, the test will not work with the dev server any longer?

Reply:
you would need to change this 
base_url = "http://localhost"
in the pyproject.toml

Reply:
Is there a way to tell it pytest without changing the file? E.g. separate profiles or a command line option for the port number? You could enter a new issue for this, if you prefer.

Reply:
pytest  --base-url http://localhost:8000
also works

Reply:
I will add it to the readme